flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -273,6 +447,32 @@ void TizenWindowEcoreWl2::RegisterEventHandlers() {
       ECORE_EVENT_MOUSE_BUTTON_DOWN,
       [](void* data, int type, void* event) -> Eina_Bool {
         auto* self = static_cast<TizenWindowEcoreWl2*>(data);
+#ifdef TV_PROFILE
+        if ((!self->pointing_device_support_ ||
+             !self->floating_menu_support_) &&
+            !self->show_unsupported_toast_) {
+          bool shown = GetPointingDeviceToastPreference();
+          if (!self->pointing_device_support_) {
+            if (!shown) {
+              // Toast popup should be called first to set up D-PAD.
+              self->ShowUnsupportedToast();
+              SetPointingDevicePreference();
+            }
+            self->SetPointingDeviceSupport();
+
+          } else if (!self->floating_menu_support_) {
+            self->SetFloatingMenuSupport();
+            if (!shown) {
+              self->ShowUnsupportedToast();
+              SetPointingDevicePreference();
+            }
+          }
+
+          self->show_unsupported_toast_ = true;
+          return ECORE_CALLBACK_PASS_ON;
```

Comment:
```suggestion
          bool shown = GetPointingDeviceToastPreference();
          if (!shown) {
            // Toast popup should be called first to set up D-PAD.
            self->ShowUnsupportedToast();
            SetPointingDevicePreference();
          }
          self->show_unsupported_toast_ = true;

          if (!self->pointing_device_support_) {
            self->SetPointingDeviceSupport();
          } else if (!self->floating_menu_support_) {
            self->SetFloatingMenuSupport();
          }
          return ECORE_CALLBACK_PASS_ON;
```
I think it can be organized like this, what do you think?

Reply:
If SetFloatingMenuSupport() is not called first, when the right button is pressed, the FloatingMenu is displayed first while the ToastPopup is loading, and then SetFloatingMenuSupport() is executed. 

Of course, even with this approach, there were cases where the FloatingMenu appeared 1-2 times out of 10. However, since we couldn't control the global event of the right-click button on the FloatingMenu, I had no choice but to call SetFloatingMenuSupport() first.

Reply:
Thank you for review. Removed duplication of Toast code.
If both are false, only SetFloatingMenuSupport() should be called.